.changeset/angry-parrots-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/react': major
```

Comment:
We shouldn't need a major for adding a dependency. I think patch is fine.

We can also merge it as a single changeset like this:

```
---
'@astrojs/react': patch
'@astrojs/svelte': patch
...
---

Resolves vite peer dependency issue for strict package managers
```

Also we're missing an update for preact here.

Reply:
I've just updated the changeset and `preact` as well. It didn't even have `vite` in devDependencies though.